src/content/guides/advanced-ingress-configuration/index.md

```diff
@@ -348,6 +348,7 @@ data:
   enable-vts-status: "true"
   error-log-level: "error"
   hsts: "false"
+  large-client-header-buffers: "4 8K"
```

Comment:
Is the `4 8K` a typo? Otherwise I'm missing something :)

Reply:
No. First number is the number and second the size.

http://nginx.org/en/docs/http/ngx_http_core_module.html#large_client_header_buffers

Reply:
Ah cool. Got it now, thx!